### PR TITLE
Dev 802 network create fields arent saveds

### PR DESCRIPTION
**Pull request by clarkjoao**

### CODE REVIEW

File: contexts/network-settings.tsx [hunk 1]
```diff
@@ -64,120 +62,199 @@ export const NetworkSettingsProvider = ({ children }) => {
...
+    newState.tokens.validated = tokensValidated;
+    newState.isSettingsValidated = [tokensLockedValidate,
+                                    detailsValidate,
+                                    githubValidate,
+                                    settingsValidated,
+                                    tokensValidated,
+    ].every(condtion=> condtion)
+
+    if(detailsValidate)
+      localStorage.setItem(`${localStorageKey}:${wallet.address}`,
```
Comment (vhcsilva) [inline contexts/network-settings.tsx:170]: Are we clearing this information when the user disconnect ? @clarkjoao

Comment (clarkjoao) [reply contexts/network-settings.tsx:170]: No, only past 2 days or after create network.

Comment (vhcsilva) [reply contexts/network-settings.tsx:170]: As we are clearing other data from localStorage on disconnect, i think we can do it for network settings data too

Comment (clarkjoao) [reply contexts/network-settings.tsx:170]: I did a double check, and when we disconnect we are cleaning every items in localstorage. 🙂

Screenshot 2022-09-28 at 21 01 28

**Review by moshmage — CHANGES_REQUESTED** [review]
decrease the debounce time from 1000 (1sec) to 500ms (half a second)

**Comment by moshmage** [comment]
@clarkjoao check the conflicts

**Pull request merged**